### PR TITLE
Update to LiteCore 3.2.0-148

### PR DESCRIPTION
**Pull request by bmeike**
Update to LiteCore 148:
- Add Replicator id to replicator creation methods
- Adjust to changes in Fleece

### CODE REVIEW

**Comment by borrrden** [comment]
Just note that I still see failing date time tests with this version of Core

**Review by borrrden — DISMISSED** [review]

**Comment by bmeike** [comment]
Yes.  I see those Date/Time failures too.   I'm not going to address them until Core gets stable.

File: common/test/java/com/couchbase/lite/ReplicatorConfigurationTest.kt [hunk 1]
```diff
@@ -71,7 +71,7 @@ class ReplicatorConfigurationTest : BaseReplicatorTest() {
         val opts = immutableConfig.connectionOptions
         assertEquals(Defaults.Replicator.HEARTBEAT, opts[C4Replicator.REPLICATOR_HEARTBEAT_INTERVAL])
         assertEquals(Defaults.Replicator.MAX_ATTEMPTS_SINGLE_SHOT - 1, opts[C4Replicator.REPLICATOR_OPTION_MAX_RETRIES])
-        assertEquals(Defaults.Replicator.MAX_ATTEMPT_WAIT_TIME, opts[C4Replicator.REPLICATOR_OPTION_MAX_RETRY_INTERVAL])
+        assertEquals(Defaults.Replicator.MAX_ATTEMPTS_WAIT_TIME, opts[C4Replicator.REPLICATOR_OPTION_MAX_RETRY_INTERVAL])
```
Comment (pasin) [inline common/test/java/com/couchbase/lite/ReplicatorConfigurationTest.kt:74]: To be correct, I think we will need to deprecate the old one.

**Review by pasin — CHANGES_REQUESTED** [review]

File: common/test/java/com/couchbase/lite/ReplicatorConfigurationTest.kt (hunk 1 above)
Comment (pasin) [reply common/test/java/com/couchbase/lite/ReplicatorConfigurationTest.kt:74]: Seem like this will get taken care as a separate PR.

**Review by pasin — APPROVED** [review]

File: common/test/java/com/couchbase/lite/ReplicatorConfigurationTest.kt (hunk 1 above)
Comment (bmeike) [reply common/test/java/com/couchbase/lite/ReplicatorConfigurationTest.kt:74]: Yeah.... Defaults.java is in (two) other repositories

**Pull request merged**